service-commands/src/setup.js

```diff
@@ -212,6 +213,25 @@ const runSetupCommand = async (
     return
   }
 
+  if (service === Service.LIBS && setupCommand === SetupCommand.UP) {
```

Comment:
@raymondjacobson I couldn't simply use `npm run dev &` in the `service-commands.json` because `exec()` waits for it even though we launch a new shell and it's backgrounded. I tried a bunch of different variants but I'm pretty sure we need to `spawn` this process to detach it from the parent.

Also, I wrapped it in `run-libs.sh` so that we could easily grep for it and kill the process group rather than trying to export a env var (which, per my other comment, isn't possible to bubble up from a child process)